eip-0027.md

```diff
@@ -60,7 +60,11 @@ We introduce two new contracts which are described in greater detail below.
 
 Miners are enforced to pay to this proxy contract. The contract is similar to the existing fee contract but with no time lock. 
 
-* We create new consensus-level checks that verify that the proper amount of re-emission tokens are going from the emission box to a miner rewards box, and also that during spending a box which is not an emission box but contains re-emission tokens (miner rewards box), the tokens are being burnt, and the same amount of nanoERG is locked by the re-emission contract. 
+* We create new consensus-level checks that verify that the proper amount of re-emission tokens are going from 
+the emission box to a miner rewards box, and also that during spending a box which 
+is not an emission box but contains re-emission tokens (miner rewards box), the tokens are being burnt, 
+and the same amount of nanoERG is locked by the re-emission contract (via a proxy pay-to-reemission contract,
+ ERGs locked by such a contract could be paid to reemission contract box (its NFT) only). 
```

Comment:
This wording is confusing. I think we can simply replace above paragraph with the below (the rest is automatically implied by the contract):

> the emission box to a miner rewards box, and also that during spending a box which 
is not an emission box but contains re-emission tokens (miner rewards box), the tokens are being burnt, 
and the same amount of nanoERG is locked in a box with the pay-to-reemission contract (aka the proxy contract).